test/functional/outputs/syslog/rfc3164_test.go

```diff
@@ -88,40 +87,37 @@ var _ = Describe("[Functional][Outputs][Syslog] RFC3164 tests", func() {
 		})
 	})
 
-	DescribeTable("configured to addLogSourceToMessage should add namespace, pod, container name", func(source string, requiresFluentd bool) {
+	DescribeTable("configured to addLogSourceToMessage should add namespace, pod, container name", func(source obs.InputType, requiresFluentd bool) {
 		if requiresFluentd && testfw.LogCollectionType != logging.LogCollectionTypeFluentd {
```

Comment:
Delete me... and/or replace with skip and //todo comment if appropriate

Reply:
I've commented the appropriate test out already with a comment above the test that needs to be fixed